charlesinto/iReporter
# #162591852 Update intervention record location

#### What does this PR do?

- Update intervention location

#### Description of Task to be completed?
- Implement JWT authentication
- Verify user token
- User should be able to update comment

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `PATCH` request to `http://localhost:7000/api/v1/interventions/:id/location`
- You should receive a successful response

#### Any background context you want to provide?
The request should be a structure like:

`{location: "string"}`

`

#### What are the relevant pivotal tracker stories?
[#162591852](https://www.pivotaltracker.com/story/show/162591852)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/26615085/49831596-87322e00-fd94-11e8-9bdb-259aea7745d6.png)






#### Questions:
N/A
